torch/testing/_comparison.py

```diff
@@ -1531,3 +1535,55 @@ def assert_close(
         check_stride=check_stride,
         msg=msg,
     )
+
+
+def assert_allclose(
+    actual: Any,
+    expected: Any,
+    rtol: Optional[float] = None,
+    atol: Optional[float] = None,
+    equal_nan: bool = True,
+    msg: str = "",
+) -> None:
+    """
+    .. warning::
+
+       :func:`torch.testing.assert_allclose` is deprecated since ``1.12``. Please use
```

Comment:
"deprecated since PyTorch 1.12 and will be removed in a future release."

Reply:
Oh, you have some optimism left :smile: